lms/migrations/versions/efcf8671f4d3_add_the_groups_table.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""Add the groups table."""
```

Comment:
I wonder if this description might be expanded to describe this table beyond "groups." It's a table for...relating an `h` service group to a `(tool_consumer_instance_guid, context_id)` unique combination, correct? There may be better places to document this in full, but expanding this docstring at least minimally couldn't hurt a future dev trying to figure out the system.